go.sum

```diff
@@ -1294,8 +1294,8 @@ kubevirt.io/controller-lifecycle-operator-sdk v0.1.1 h1:dqURkr5Q5f4qJM7uLHL0akFx
 kubevirt.io/controller-lifecycle-operator-sdk v0.1.1/go.mod h1:M+UuZsp90mmKUVGb5zXDZtIIY1noY5pJk9i2YCQwSnk=
 kubevirt.io/qe-tools v0.1.6 h1:S6z9CATmgV2/z9CWetij++Rhu7l/Z4ObZqerLdNMo0Y=
 kubevirt.io/qe-tools v0.1.6/go.mod h1:PJyH/YXC4W0AmxfheDmXWMbLNsMSboVGXKpMAwfKzVE=
-libvirt.org/libvirt-go v6.5.0+incompatible h1:7hrVIm1onDKjIYpKyWqAtF6+ugYOVSnyaWnNpn+8tLI=
-libvirt.org/libvirt-go v6.5.0+incompatible/go.mod h1:CPoljLoiC2aEw+62g1rZXl2oXAJaNsrq4YCSmJOELek=
+libvirt.org/libvirt-go v7.0.0+incompatible h1:Ebw+ZbbsCqB8ok4UmVpB9YhE3axm9nQGiuotGerau7Q=
+libvirt.org/libvirt-go v7.0.0+incompatible/go.mod h1:CPoljLoiC2aEw+62g1rZXl2oXAJaNsrq4YCSmJOELek=
```

Comment:
We need to move to qemu 5.2.0 at the same time we move to this libvirt version

Reply:
Thanks @stu-gott